### PR TITLE
Obs rep ocp

### PR DESCRIPTION
**Pull request by anandrajak1**
# Description
OpenShift 4.13 support  for replication and observability 

# GitHub Issues
List the GitHub issues impacted by this PR:

| GitHub Issue # |
| -------------- |
| https://github.com/dell/csm/issues/724|

# Checklist:

- [x] Have you run a grammar and spell checks against your submission?
- [x] Have you tested the changes locally?
- [x] Have you tested whether the hyperlinks are working properly?

### CODE REVIEW

**Review by santhoshatdell — APPROVED** [review]

**Review by harshaatdell — DISMISSED** [review]

**Review by bharathsreekanth — DISMISSED** [review]

File: content/docs/observability/_index.md [hunk 1]
```diff
@@ -7,14 +7,14 @@ Description: >
 ---
 
  [Container Storage Modules](https://github.com/dell/csm) (CSM) for Observability is part of the open-source suite of Kubernetes storage enablers for Dell products.
- 
- It is an OpenTelemetry agent that collects array-level metrics for Dell storage so they can be scraped into a Prometheus database. With CSM for Observability, you will gain visibility not only on the capacity of the volumes/file shares you manage with Dell CSM CSI (Container Storage Interface) drivers but also their performance in terms of bandwidth, IOPS, and response time. 
- 
+
+ It is an OpenTelemetry agent that collects array-level metrics for Dell storage so they can be scraped into a Prometheus database. With CSM for Observability, you will gain visibility not only on the capacity of the volumes/file shares you manage with Dell CSM CSI (Container Storage Interface) drivers but also their performance in terms of bandwidth, IOPS, and response time.
```
Comment (sharont58) [inline content/docs/observability/_index.md:11]: I don't understand the term "scraped". Can you find a different term?

File: content/docs/observability/_index.md [hunk 2]
```diff
@@ -7,14 +7,14 @@ Description: >
...
 
 Metrics data is collected and pushed to the [OpenTelemetry Collector](https://github.com/open-telemetry/opentelemetry-collector), so it can be processed, and exported in a format consumable by Prometheus. SSL certificates for TLS between nodes are handled by [cert-manager](https://github.com/jetstack/cert-manager).
 
-CSM for Observability is composed of several services, each living in its own GitHub repository, that can be installed following one of the four deployments we support [here](deployment). Contributions can be made to this repository or any of the CSM for Observability repositories listed below. 
+CSM for Observability is composed of several services, each living in its own GitHub repository, that can be installed following one of the four deployments we support [here](deployment). Contributions can be made to this repository or any of the CSM for Observability repositories listed below.
```
Comment (sharont58) [inline content/docs/observability/_index.md:17]: Instead of "living" use "residing"

File: content/docs/observability/_index.md (hunk 1 above)
Comment (anandrajak1) [reply content/docs/observability/_index.md:11]: "exported" how about it?

Comment (sharont58) [reply content/docs/observability/_index.md:11]: Yes, that's good. :)

**Review by rajkumar-palani — APPROVED** [review]
lgtm

**Review by santhoshatdell — APPROVED** [review]

**Pull request merged**